app/src/pages/prompt/promptCodeSnippets.tsx

```diff
@@ -337,15 +432,53 @@ export const promptCodeSnippets: Record<
   },
 };
 
-export const mapPromptToSnippet = ({
+export const promptClientCodeSnippets: Record<
+  string,
+  Record<string, PromptToClientSnippetFn>
+> = {
+  python: {
+    openai: (prompt) => {
+      const config = languageConfigs.python.openai;
+      return config.clientTemplate({
+        tab: TAB,
+        verionId: prompt.versionId,
+      });
+    },
+    anthropic: (prompt) => {
+      const config = languageConfigs.python.openai;
+      return config.clientTemplate({
+        tab: TAB,
+        verionId: prompt.versionId,
```

Comment:
this should be called versionId across the file

Reply:
I need cspell ;)